housekeeper/store/filters/archive_filters.py

```diff
@@ -31,19 +32,28 @@ def filter_by_retrieval_task_id(archives: Query, task_id: int, **kwargs) -> Quer
     return archives.filter(Archive.retrieval_task_id == task_id)
 
 
+def filter_by_retrieved_before(archives: Query, date: datetime, **kwargs) -> Query:
```

Comment:
```suggestion
def filter_by_retrieved_before(archives: Query, retrieved_before: datetime, **kwargs) -> Query:
```
If we get another filter that requires a date it would be a bit tricky with passing that argument if both were named 'date' also it would be confusing to what filter the date would apply.

Reply:
True, good catch!